### ISSUE TITLE
Need AFK kick timer

### ISSUE THREAD

**Issue by metalgearsloth** [space-wizards/space-station-14#10577]
People are sitting in rounds doing nothing to pad their role times and taking up a slot.

AFKSystem issues events so all you need to do is have some kind of timer that accumulates for AFK people (e.g. 5-10 minutes) and kicks them after sending a warning.

**Comment by Visne** [comment]
As far as I know role timers don't accumulate if you are AFK.

**Comment by metalgearsloth** [comment]
They don't but people also occupy a slot (when we cap a lot) attempting it.

**Comment by Visne** [comment]
yikes

**Comment by Renlou** [comment]
Yep, afk auto kick would be great. I myself have sit in lobby afk without knowing only to find out later on and felt bad cuz lizard is packed.
